<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtrasDialog</class>
 <widget class="QDialog" name="ExtrasDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ekstralar</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Ekstralar</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>301</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ButtonsLayout">
    <item>
     <widget class="QPushButton" name="SmallPopCorn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Mısır Küçük</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MediumPopCorn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Mısır Orta</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="LargePopCorn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Mısır Büyük</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SmallCola">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Kola Küçük</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MediumCola">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Kola Orta</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="LargeCola">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Kola Büyük</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Frigo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Alaska Frigo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>341</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="BillLayout"/>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>510</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tutar : </string>
   </property>
  </widget>
  <widget class="QLabel" name="BillOutput">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>70</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Müşteri Koltuk No:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>160</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Hesaba Ekle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>340</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Ekelemyi Geri Al</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
